third_party/ziplain/ziplain.cc

```diff
@@ -0,0 +1,247 @@
+#include "ziplain.h"
+
+#include <endian.h>
+#include <string.h>
+#include <zlib.h>
+
+#include <cstdio>
+#include <memory>
+#include <vector>
+
+namespace ziplain {
+ByteSource MemoryByteSource(std::string_view input) {
```

Comment:
maybe as part of our import efforts to this project, we should `sed 's/std::string_view/absl::string_view/'` ?

Reply:
Done